security-council-rehearsals/templates/r2-remove-signer/justfile

```diff
@@ -26,7 +26,7 @@ simulate hdPath='0':
 
   forge build
   forge script ./SignFromJson.s.sol:SignFromJson \
-    --rpc-url ${rpcUrl} \
+    --rpc-url ${ETH_RPC_URL} \
```

Comment:
I didn't fully check but the value here was not read previously already by the `ETH_RPC_URL`?

Reply:
Oh okay, I see just above you are also reading the toml file.